packages/gatsby/src/cache-dir/production-app.js

```diff
@@ -18,7 +18,9 @@ window.___emitter = emitter
 import pages from "./pages.json"
 import ComponentRenderer from "./component-renderer"
 import asyncRequires from "./async-requires"
+import syncRequires from "./sync-requires"
```

Comment:
sync-requires is just for development. It creates one giant bundle which is fine for dev but not for prod. `asyncRequires` has all the layout bundles.